app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryShowActivity.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+package ch.epfl.sdp.drone3d.ui.mission
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.drone3d.R
+import ch.epfl.sdp.drone3d.map.MapboxMissionDrawer
+import ch.epfl.sdp.drone3d.map.MapboxUtility
+import com.mapbox.mapboxsdk.Mapbox
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.mapboxsdk.maps.MapView
+import com.mapbox.mapboxsdk.maps.Style
+
+class ItineraryShowActivity : AppCompatActivity() {
```

Comment:
We should agree on a naming convention between mission / Itinerary

Reply:
Yes that's right. I named it itinerary because the activity to create a mission was also called ItineraryCreateActivity. But mission might me more appropriate